src/server/controller/user/acceptInvitation.ts

```diff
@@ -23,7 +23,7 @@ export const acceptInvitation = async (
 
   return client.tx(async (t) => {
     if (UserRoles.isInvitationExpired(userRole)) throw new Error('login.invitationExpired')
-    if (!UserRoles.isInvitationPending(userRole)) throw new Error('login.alreadyAcceptedInvitation')
+    if (userRole.acceptedAt !== null) throw new Error('login.alreadyAcceptedInvitation')
```

Comment:
The dates seems to be considered empty by Objects.isEmpty
![Screenshot 2024-02-16 at 2 33 49 PM](https://github.com/openforis/fra-platform/assets/41337901/d720e20b-4868-4bd1-b626-91c7df7da088)